t/lib/Test/PromAlertProxy.pm

```diff
@@ -1,7 +1,11 @@
 package Test::PromAlertProxy;
 
-use warnings;
+use 5.028;
 use strict;
+use experimental qw(signatures);
+
+use Test::PromAlertProxy::Target;
+use Test::PromAlertProxy::CrashTarget;
 
 sub now { 1558324375 };
```

Comment:
you gotta use your birthday as test timestamps, man, it's the law

Reply:
But then Major League Baseball will know where I live.